src/prerecorded/options/mod.rs

```diff
@@ -0,0 +1,337 @@
+use serde::{ser::SerializeSeq, Serialize};
+
+#[derive(Debug)]
+pub struct OptionsBuilder<'a> {
+    model: Option<Model<'a>>,
+    version: Option<&'a str>,
+    language: Option<Language>,
+    punctuate: Option<bool>,
+    profanity_filter: Option<bool>,
+    redact: Vec<Redact>,
+    diarize: Option<bool>,
+    ner: Option<bool>,
+    multichannel: Option<bool>,
+    alternatives: Option<usize>,
+    numerals: Option<bool>,
+    search: Vec<&'a str>,
+    callback: Option<&'a str>,
+    keywords: Vec<&'a str>,
+    utterances: Option<Utterances>,
+    tag: Option<&'a str>,
+}
+
+#[derive(Debug)]
+pub enum Model<'a> {
+    General,
+    Meeting,
+    Phonecall,
+    Voicemail,
+    Finance,
+    Conversational,
+    Video,
+    CustomId(&'a str),
+}
+
+#[derive(Debug, Clone, Copy)]
+#[non_exhaustive]
+#[allow(non_camel_case_types)]
+pub enum Language {
+    zh_CN,
+    zh_TW,
+    nl,
+    en_US,
+    en_AU,
+    en_GB,
+    en_IN,
+    en_NZ,
+    fr,
+    fr_CA,
+    de,
+    hi,
+    id,
+    it,
+    ja,
+    ko,
+    pt,
+    pr_BR,
+    ru,
+    es,
+    es_419,
+    sv,
+    tr,
+    uk,
+}
```

Comment:
This is good for enumerating the languages we support, but now we need to keep this up to date whenever a new language gets added.  Is it worth adding an `Other(String)` variant, so we don't end up holding users back?  Or is that too confusing?  Maybe document it:

```rust
enum Language {
    ...
    tr,
    uk,
    /// We try to keep this list of languages up-to-date, but Deepgram moves fast and so do you.  
    /// If you need to transcribe languages that are supported by Deepgram, but aren't yet listed above, 
    /// you can supply a language code here.
    Other(String),
}
```

Reply:
Implemented in 07e3106542a8202f26d5e2a6fe7c4c190d897107.